edb/server/protocol/ai_ext.py

```diff
@@ -354,11 +355,13 @@ async def _ext_ai_fetch_active_models(
     result = []
     if models:
         for model in models:
-            result.append((
-                int.from_bytes(model[0], byteorder="big", signed=True),
-                model[1].decode("utf-8"),
-                model[2].decode("utf-8"),
-            ))
+            result.append(
+                (
+                    int.from_bytes(model[0], byteorder="big", signed=True),
+                    model[1].decode("utf-8"),
+                    model[2].decode("utf-8"),
+                )
+            )
```

Comment:
@diksipav I'm not sure what happened here -- a code formatter tool? but please revert all changes that are pure syntax/whitespace changes.

Reply:
Hey, I fixed formatting. @elprans, @dnwpark can you pls take a look at this PR, I'll be on vacation next week. But would like to move this forward, and be able to add new things in new PRs.